c/tests/test_tables.c

```diff
@@ -5078,6 +5078,8 @@ test_dump_unindexed_with_options(tsk_flags_t tc_options)
     parse_edges(single_tree_ex_edges, &tables.edges);
     CU_ASSERT_EQUAL_FATAL(tables.edges.num_rows, 6);
     CU_ASSERT_FALSE(tsk_table_collection_has_index(&tables, 0));
+    ret = tsk_table_collection_build_index(&tables, 0);
+    CU_ASSERT_EQUAL(ret, 0);
     ret = tsk_table_collection_dump(&tables, _tmp_file_name, 0);
     CU_ASSERT_EQUAL_FATAL(ret, 0);
```

Comment:
I think what we'd want to test is that we *don't* have an index after loading when we write a table collection without an index. So, delete the new lines and change the assert to FALSE below.

Reply:
i